src/api.rs

```diff
@@ -9,17 +9,27 @@ pub(crate) async fn route(
     req: hyper::Request<hyper::Body>,
     tx: mpsc::Sender<wasm::EvalRequest>,
 ) -> Result<hyper::Response<hyper::Body>, hyper::Error> {
-    match (req.method(), req.uri().path()) {
-        (&Method::POST, "/validate") => handle_post_validate(req, tx).await,
+    match req.method() {
+        &Method::POST => {
+            let path = String::from(req.uri().path());
+            if path.starts_with("/validate/") {
+                handle_post_validate(req, path.trim_start_matches("/validate/").to_string(), tx)
```

Comment:
With there being separate endpoints for each policy, does this mean that the chimera controller will be creating a separate webhookconfig for every policy pointing to each `/validate/<id>` endpoint? Then kubernetes will have to make a round trip for every policy on every pod request. Why do we not just point the webhook to `/validate` and iterate over every configured policy on this end?

Reply:
Yes, you are correct. The chimera controller will create multiple `ValidatingWebhookConfigurations` objects, one per policy that is defined by the user. The configuration objects will point to the same `policy-server` instance, just to different endpoints.

It's also true that Kubernetes will have to make a round trip for every policy that has to be validated, but:

  * Different policies could process different kind of Kubernetes resources, or could be interested only to a subset of actions. This is expressed inside of the `ValidatingWebhookConfigurations`, Kubernetes knows better which of the chimera policies are relevant for a certain incoming request.
  * Maybe a request has to be evaluated by `N` policies. The roundtrip happens only when all the `N-1` policies have been accepted. An earlier rejection halts the full chain of validation.
  * Default timeout for a webhook call is 10 seconds for webhooks registered created using `admissionregistration.k8s.io/v1` (I took that from [here](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)), but this is tunable by the user. If we iterate over all the policies we might exceed that timeout.

Overall, I would not want `policy-server` to deal with all the nuances of iterating over all the policies. Kubernetes has already code for it, it's also a supported use case to have multiple webhook endpoints registered... So, I would start with something simple, we can always change that later on if needed